controllers/usersController.js

```diff
@@ -1,21 +1,21 @@
-const express = require('express');
-const { getAllUsers, getUserById } = require('../queries/usersQueries');
+const express = require("express");
+const { getAllUsers, getUserById } = require("../queries/usersQueries");
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
+usersController.get("/", async (req, res) => {
+  users = (await getAllUsers()) || [];
```

Comment:
The upstream repo has a mistake in this line of code: there should be `const` or `let` before users.

I think you can simplify this line to 
```
const users = await getAllUsers();
```
`getAllUsers()` calls `db.any()` and that method always returns an array (never returns null). So I don't think you need the `|| []` at the end.